### PR TITLE
Add attributes section

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

File: docs/developer/attributes.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Attributes
+sidebar_label: Attributes
+---
+
+## Introduction
+
+Setting up attributes will allow your users to filter your products catalogue.
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:8]: Setting up attributes allows users to filter product catalogues.

Comment (karolkielecki) [inline docs/developer/attributes.mdx:8]:
```suggestion
Setting up attributes allows users to filter product catalogues.
```

File: docs/developer/attributes.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Attributes can be used for
+- [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
+- [Pages](developer/api-reference/objects/page.mdx)
+
+## Use cases
+
+1. Your store sells Mugs in many colors and sizes. This means you will need:
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:16]: Your store sells mugs in many colors and sizes. In this example, you can set up the following attributes:

Comment (karolkielecki) [inline docs/developer/attributes.mdx:16]:
```suggestion
1. Your store sells mugs in many colors and sizes. In this example, you can set up the following attributes:
```

File: docs/developer/attributes.mdx (hunk 1 above)
Comment (patrys) [inline docs/developer/attributes.mdx:8]:
```suggestion
Properly configured attributes provide crucial information about your products and allow users to find the ones they are looking for quicker.
```

File: docs/developer/attributes.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- Color attribute (red, black, etc.)
+- Size attribute (Small, Big)
+
+2. Your shop has many Pages which should render with different styles. To achieve that, you can create attribute named `Template` with type `PAGE_TYPE`. As values you could set: `Support`, `Blog`, `Promos`. Then in storefront code you could load proper css template based on page `Template` value.
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:20]: Your shop has many pages with different rendering styles. You can create an attribute called `Template` with a `PAGE_TYPE` type. You can set the following values: `Support`, `Blog`, `Promos`. Then you can upload a proper CSS template based on the `Template` value in the storefront code.

Comment (karolkielecki) [inline docs/developer/attributes.mdx:20]:
```suggestion
2. Your shop has many pages with different rendering styles. You can create an attribute called `Template` with a `PAGE_TYPE` type. You can set the following values: `Support`, `Blog`, `Promos`. Then you can upload a proper CSS template based on the `Template` value in the storefront code. 
```

File: docs/developer/attributes.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+## Permissions
+
+To edit and create Attributes user is required to have permissions:
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:24]: To edit and create attributes, a user needs to have permissions:

Comment (karolkielecki) [inline docs/developer/attributes.mdx:24]:
```suggestion
To edit and create attributes, a user needs to have permissions:
```

File: docs/developer/attributes.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`
+- `MANAGE_PAGE_TYPES_AND_ATTRIBUTES`
+
+Assigning Attrubutes to Products and Pages does not require additional permission besides `MANAGE_PRODUCTS` and `MANAGE_PAGES`
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:28]: To assign attributes to products and pages, a user needs only the following permissions: `MANAGE_PRODUCTS` and `MANAGE_PAGES`.

Comment (karolkielecki) [inline docs/developer/attributes.mdx:28]:
```suggestion
To assign attributes to products and pages, a user needs only the following permissions: `MANAGE_PRODUCTS` and `MANAGE_PAGES`.
```

File: docs/developer/attributes.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+## Filtering by attributes
+
+For filtering by attributes you can use their `slug` name. This way you can be sure that query parameters can be encoded in the URL and will be the same in all languages.
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:32]: A user can use a `slug`  to filter products by attributes. The query parameters can be encoded in URL and they will be the same in all languages.

Comment (karolkielecki) [inline docs/developer/attributes.mdx:32]:
```suggestion
A user can use a `slug`  to filter products by attributes. The query parameters can be encoded in URL and they will be the same in all languages.
```

File: docs/developer/attributes.mdx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+Example of [`products`](developer/api-reference/queries/products.mdx) query:
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:34]: An example of [`products`](developer/api-reference/queries/products.mdx) query:

Comment (karolkielecki) [inline docs/developer/attributes.mdx:34]:
```suggestion
An example of [`products`](developer/api-reference/queries/products.mdx) query:
```

File: docs/developer/attributes.mdx [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+sidebar_label: Attributes
+---
+
+## Introduction
+
+Setting up attributes will allow your users to filter your products catalogue.
+
+Attributes can be used for
+- [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
+- [Pages](developer/api-reference/objects/page.mdx)
```
Comment (fowczarek) [inline docs/developer/attributes.mdx:12]: @IKarbowiak  working on attributes for categories.

Comment (fowczarek) [reply docs/developer/attributes.mdx:12]: https://github.com/mirumee/saleor/tree/feature/category-attributes

**Review by fowczarek — APPROVED** [review]

File: docs/developer/attributes.mdx [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+## Use cases
+
+1. Your store sells Mugs in many colors and sizes. This means you will need:
+- Color attribute (red, black, etc.)
+- Size attribute (Small, Big)
```
Comment (IKarbowiak) [inline docs/developer/attributes.mdx:18]: As examples of color attributes start with lowercase, size examples should too.
```suggestion
- Size attribute (small, big)
```

File: docs/developer/attributes.mdx (hunk 4 above)
Comment (IKarbowiak) [reply docs/developer/attributes.mdx:24]: Maybe:
```suggestion
To edit and create attributes, a user needs to have the following permissions:
```

**Review by IKarbowiak — APPROVED** [review]

File: docs/developer/attributes.mdx [hunk 10]
```diff
@@ -5,33 +5,33 @@ sidebar_label: Attributes
...
-Assigning Attrubutes to Products and Pages does not require additional permission besides `MANAGE_PRODUCTS` and `MANAGE_PAGES`
+To assign attributes to products and pages, a user needs only the following permissions: `MANAGE_PRODUCTS` and `MANAGE_PAGES`.
 
 ## Filtering by attributes
 
-For filtering by attributes you can use their `slug` name. This way you can be sure that query parameters can be encoded in the URL and will be the same in all languages.
+A user can use a `slug`  to filter products by attributes. The query parameters can be encoded in URL and they will be the same in all languages.
 
-Example of [`products`](developer/api-reference/queries/products.mdx) query:
+An example of [`products`](developer/api-reference/queries/products.mdx) query:
```
Comment (karolkielecki) [inline docs/developer/attributes.mdx:34]: An example of a [`product's`](developer/api-reference/queries/products.mdx) query:

Comment (karolkielecki) [inline docs/developer/attributes.mdx:34]:
```suggestion
An example of a [`product's`](developer/api-reference/queries/products.mdx) query:
```

Comment (patrys) [reply docs/developer/attributes.mdx:34]: @karolkielecki It's supposed to say "products," not "product's."

```suggestion
An example of a [`products`](developer/api-reference/queries/products.mdx) query:
```

File: docs/developer/attributes.mdx (hunk 8 above)
Comment (patrys) [reply docs/developer/attributes.mdx:12]: Can we sort those lists alphabetically?

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**